design_system/src/commonMain/kotlin/net/thechance/mena/designsystem/presentation/theme/theme/MenaTheme.kt

```diff
@@ -15,22 +18,28 @@ import net.thechance.mena.designsystem.presentation.theme.spacing.Spacing
 import net.thechance.mena.designsystem.presentation.theme.typography.LocalTypography
 import net.thechance.mena.designsystem.presentation.theme.typography.Typography
 import net.thechance.mena.designsystem.presentation.theme.typography.createThemeTypography
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.InternalResourceApi
 
+@OptIn(InternalResourceApi::class, ExperimentalResourceApi::class)
 @Composable
-fun MenaTheme(content: @Composable () -> Unit) {
+fun MenaTheme(
+    language: String = "en",
+    content: @Composable () -> Unit,
+) {
     val colorScheme = LightColorScheme
     val typography = createThemeTypography()
-
+    val layoutDirection = if (language == "ar") LayoutDirection.Rtl else LayoutDirection.Ltr
```

Comment:
I would make this in a remember with language key to avoid unnecessary recomposition

 ```suggestion
     val layoutDirection = remember(language) {
      if (language == "ar") LayoutDirection.Rtl else LayoutDirection.Ltr
     }
```

Reply:
done please check

Reply:
I think this one is not done yet??